README.md

```diff
@@ -1,17 +1,40 @@
-# mailpile
+### mailpile
+--------
+
+A free & open modern, fast email client with user-friendly encryption and privacy features  https://mailpile.is !
 
 ### Installing mailpile
- The following commands will setup a docker installation for Mailpile:
+-----------------------
+
+ ###### The following commands will setup a docker installation for Mailpile:
 
-    Installation First install ds: https://github.com/docker-scripts/ds#installation
-    Clone the scripts from github: git clone https://github.com/nkwate-kazim/mailpile.git
-    Move the script into /opt/docker-scripts/: cp mailpile /opt/docker-scripts/
-    cd /opt/docker-scripts/
-    ds init mailpile @mailpile
-    source ds cd @mailpile   # (or: cd /var/ds/mailpile/)
-    nano settings.sh
-    ds build
-    ds create
-    ds config 
-    Add to wsproxy: ds wsproxy add mailpile.example.org
-    Navigate to mailpile.example.org:33411
+   1. Installation First install ds: https://github.com/docker-scripts/ds#installation
+   
+   2. Clone the scripts from github: git clone https://github.com/.../mailpile.git
+   
+   3. Move the script into /opt/docker-scripts/: 
+   ```
+      cp mailpile /opt/docker-scripts/
+   ```
+   4. cd /opt/docker-scripts/
+   
+   5. Initialize a container directory for mailpile
+   ```
+      ds init mailpile @mailpile
+   ```
+   6. Initialize and fix the settings:
+   ```
+      source ds cd @mailpile   # (or: cd /var/ds/mailpile/)
+      nano settings.sh
+   ```
+   
+   7. Build image, create the container and configure it:
+      ```  
+      ds build
+      ds create
+      ds config 
+      ```
+   
+   9. Add to wsproxy: ds wsproxy add mailpile.example.org
+   
+   11. Navigate to mailpile.example.org:33411
```

Comment:
This is not related to the format:
If you use `wsproxy`, then you should not need to specify that port (`:33411`) after the domain. This means that the configuration of your container needs some fixing.